### ISSUE TITLE
Alt Title

### ISSUE THREAD

**Issue by DomBouman** [zachlehman99/Lehman-Buffet#2]
You forgot to add Alt title behind the images to show the user what image they are looking at.

**Comment by zachlehman99** [comment]
Yup, points were taken off for that, but thank you for the information.
